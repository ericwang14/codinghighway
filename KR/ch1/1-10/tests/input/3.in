

He	l				loWoRl

d		\