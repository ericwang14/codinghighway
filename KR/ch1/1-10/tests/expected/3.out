

H\be\tl\t\t\t\tlo\bW\boR\bl

d\b\t\t\\